docs/glossary.md

```diff
@@ -11,3 +11,19 @@
 - **Game**: (AKA Game XML) refers to a specific XML within a map
 - **Map**: collection of files, eg: images, polygons text, properties, and at least one game XML
 
+## Battle Terms:
+- **Roll**: How many dice can the unit roll.
+- **Strength**: The number on the dice that determines whether a hit was made. Any number less than or equal to the number is a hit.
+- **Round**: A battle round includes both sides firing at each other. A battle can consist of multiple rounds. During and at the end of each round, casualties are removed from the game and can not participate anymore.
+- **Phase**: A battle phase is subpart of a battle round where specific rules are in play that determine what the units can do. Some phases allow the units to retreat. Other phases allow the units to fire at each other.
+- **General Phase**: This phase is the default phase for units to fire. The attacker fires first and then the defender retaliates. Any casualties that the attacker hits can retaliate. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
```

Comment:
@panther2 , @Cernelius , can you weigh in if "general phase" is a term you would use and if this definition fits your understanding of that battle phase?

Reply:
I would rather call it "General Combat Phase" or only "General Combat"

Reply:
I renamed it to "General Combat Phase".

Reply:
My 2 cents, 'combat phase' without qualifier is the 'general combat phase'.

I'd opt for either:
- combat phase
or
- general combat

The latter being a phase of combat I think is implied by the qualifier of 'combat', a battle is the whole, combats are typically phases within a battle (or phases within a combat)